RefactoringPracticeI/GameInventory/Inventory/HandlerAgedBrie.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace RefactoringExerciseI.Inventory
+{
+    public class HandlerAgedBrie : IHandlerItem 
+    {
+        private IHandlerItem? _nextHandler;
+
+        public IHandlerItem? NextHandler { get; }
+
+        public void SetNext(IHandlerItem NextHandler)
+        {
+            _nextHandler = NextHandler;
+        }
+
+        public void UpdateQuality (Item item)
+        {
+            if(item == null)
+               return;
+
+            if(item.Name == ItemNames.AgedBrie) {
+                if(item.Quality < 50)
```

Comment:
Subjective: 50 appears to be a magic number, and its being used in 2 locations, could it be moved to a constant? Seems to be the maximum quality an item can have.